src/components/FormInputItem.tsx

```diff
@@ -105,24 +127,26 @@ export function FormInputItem<S extends Record<N, State>, N extends keyof S>({
             className={`custom-file-input ${fileInputClass}`}
             onChange={handleFileChange}
           />
-          <label htmlFor={fileInputName} className={`custom-file-label ${state.mode === 'file' ? 'selected' : ''}`}>
+          <label
+            htmlFor={fileInputName}
+            className={`custom-file-label ${state.mode === 'file' ? 'selected' : ''}`}
+          >
             {state.mode === 'file'
-              ? state.status === 'loading' ? 'Loading...' : state.file.name
-              : fileLabel
-            }
+              ? state.status === 'loading'
+                ? 'Loading...'
+                : state.file.name
+              : fileLabel}
```

Comment:
Looks like its not turned on via ESLint, but I'm a fan of having no nested ternary expressions.  Nested ternary expressions can be confusing to understand.

[See the ESLint rule here which includes suggestions for alternatives (i.e. using `if...else`)](https://eslint.org/docs/rules/no-nested-ternary)

Reply:
That makes sense. Should I enable that ESLint rule too?

Reply:
Its worth automatically checking if we can, so I'd say yes